src/v/cluster/compat_backend.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "cluster/commands.h"
+#include "cluster/errc.h"
+
+#include <seastar/core/future.hh>
+
+namespace cluster {
+class compat_backend {
+public:
+    compat_backend() noexcept {}
+
```

Comment:
A comment here explaining why this is a no-op would be nice.

Reply:
Yep, added comments to this method+class.